### DISCUSSION TITLE
fs.readFileSync issue

### DISCUSSION THREAD

**Discussion by vizakenjack** [arthelokyo/astrowind#257] [Q&A]
First of all, thanks for your theme! Its awesome. Yesterday I have upgraded my project to the latest release @onwidget/astrowind@1.0.0-beta.3

It works fine, however I have an issue with my react components.

I have this error in a console:

      CitiesList.4e465334.js:1 Error reading config: TypeError: bl.readFileSync is not a function
          at CitiesList.4e465334.js:1:63921

It points to src/utils/config.ts:

      config = yaml.load(fs.readFileSync('src/config.yaml', 'utf8'))

I had to add try...catch to this block to reanimate my react components, but not sure if it's a good solution. 

Also, I have SSR enabled if that matters. This is my first JS/TS project, so I was't aware that this template doesn't officially support SSR (according to this issue https://github.com/onwidget/astrowind/discussions/194)

**Comment by prototypa** [comment]
Hi @vizakenjack 

What you say is very important. I'm going to check that out right now.

About SSR: Part of the blog does not work with SSR. It is only intended, for now, to work with the `"static"` or `"hybrid"` output settings. But I am going to check, if apart from that it is affecting the way the settings are loaded.

Thanks, I'll let you know soon.
